website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -29,6 +29,8 @@ A metric is a timeseries aggregation over a <Term id="table" /> that supports ze
 
 In v1.0, dbt supports metric definitions as a new node type. Like [exposures](exposures), metrics participate in the dbt DAG and can be expressed in YAML files. By defining metrics in dbt projects, you encode crucial business logic in tested, version-controlled code. Further, you can expose these metrics definitions to downstream tooling, which drives consistency and precision in metric reporting.
 
+For more information on query the metrics that are defined, please reference the [dbt_metrics package.](https://github.com/dbt-labs/dbt_metrics)
```

Comment:
👍  Definitely think it's more than appropriate to prominently link to the `dbt-metrics` package from this page

[Blocking] I think I know what you're saying here, but the wording is a bit tricky. Do you mean:
- "For more information on querying metrics that are defined in your dbt project"
- "For more information on how metrics queries are defined"
?

Reply:
Resolved!